src/convnets/mobilenet.jl

```diff
@@ -27,37 +27,38 @@ function mobilenetv1(width_mult, config;
                      inchannels = 3,
                      nclasses = 1000,
                      fcsize = 1024)
-  layers = []
-  for (dw, outch, stride, nrepeats) in config
-    outch = Int(outch * width_mult)
-    for _ in 1:nrepeats
-      layer = dw ? depthwise_sep_conv_bn((3, 3), inchannels, outch, activation;
-                                         stride = stride, pad = 1) :
-                   conv_bn((3, 3), inchannels, outch, activation; stride = stride, pad = 1)
-      append!(layers, layer)
-      inchannels = outch
+    layers = []
+    for (dw, outch, stride, nrepeats) in config
+        outch = Int(outch * width_mult)
+        for _ in 1:nrepeats
+            layer = dw ?
+                    depthwise_sep_conv_bn((3, 3), inchannels, outch, activation;
+                                          stride = stride, pad = 1) :
+                    conv_bn((3, 3), inchannels, outch, activation; stride = stride, pad = 1)
+            append!(layers, layer)
+            inchannels = outch
+        end
     end
-  end
 
-  return Chain(Chain(layers),
-               Chain(GlobalMeanPool(),
-                     MLUtils.flatten,
-                     Dense(inchannels, fcsize, activation),
-                     Dense(fcsize, nclasses)))
+    return Chain(Chain(layers),
+                 Chain(GlobalMeanPool(),
+                       MLUtils.flatten,
+                       Dense(inchannels, fcsize, activation),
+                       Dense(fcsize, nclasses)))
 end
 
 const mobilenetv1_configs = [
-#     dw,    c, s, r
-  (false,   32, 2, 1),
-  ( true,   64, 1, 1),
-  ( true,  128, 2, 1),
-  ( true,  128, 1, 1),
-  ( true,  256, 2, 1),
-  ( true,  256, 1, 1),
-  ( true,  512, 2, 1),
-  ( true,  512, 1, 5),
-  ( true, 1024, 2, 1),
-  ( true, 1024, 1, 1)
+    #     dw,    c, s, r
```

Comment:
This slightly misaligned the "columns" for the comment which can be fixed by removing space in front of `dw`.